complete/src/main/java/price/checker/Application.java

```diff
@@ -0,0 +1,14 @@
+package price.checker;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+@SpringBootApplication
+@EnableJpaRepositories("price.checker.domain.rpsy")
```

Comment:
I think spring should be able to pick up the repository without this annotation, because @SpringBootApplication already contains @EnableAutoConfiguration, which should find it. But I'm not positive on that. See if it works without it.